env.sh

```diff
@@ -15,6 +15,8 @@
 export X_CSI_VXFLEXOS_THICKPROVISION=false
 export X_CSI_VXFLEXOS_ENABLESNAPSHOTCGDELETE="true"
 export X_CSI_VXFLEXOS_ENABLELISTVOLUMESNAPSHOTS="true"
+#Uncomment X_CSI_QUOTA_ENABLED if you want to enable quota for nfs volume
```

Comment:
Remove the commented code, if not needed.

Reply:
This code is needed